project_time_range/tests/test_project_project.py

```diff
@@ -50,3 +50,63 @@ def test_idealCanBeNone(self):
         })
         assert self.project_a.max_hours == 0
         assert self.project_a.min_hours == 0
+
+
+class TestProjectBudgetFields(common.SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.project_a = cls.env["project.project"].create({"name": "Project A"})
+        cls.task_a = cls.env["project.task"].create(
+            {
+                "name": "Task Template A",
+                "is_template": True,
+                "project_id": cls.project_a.id,
+                "min_hours": 1,
+                "planned_hours": 2,
+                "max_hours": 4,
+            }
+        )
+        cls.task_b = cls.env["project.task"].create(
+            {
+                "name": "Task Template B",
+                "is_template": True,
+                "project_id": cls.project_a.id,
+                "min_hours": 8,
+                "planned_hours": 16,
+                "max_hours": 32,
+            }
+        )
+
+    def test_project_budget_fields_sum(self):
+        assert self.project_a.min_hours == 9
+        assert self.project_a.planned_hours == 18
+        assert self.project_a.max_hours == 36
+
+    def test_project_tracking_message_for_task_template_add(self):
+        task_c = self.env["project.task"].create(
```

Comment:
Codacy found an issue: [local variable 'task_c' is assigned to but never used (F841)](https://app.codacy.com/gh/Numigi/odoo-project-addons/pullRequest?prid=6260167)